.travis/README.md

```diff
@@ -0,0 +1,185 @@
+
+### Writing the Deployment Script
+Next, create a deploy script to install and run the Flyctl command-line tool on the Travis CI virtual environment after the build succeeds. Create a new directory called `scripts` with a file called `deploy.sh` inside:
+
+```
+mkdir scripts && touch scripts/deploy.sh
+```
+
+Open the newly created `deploy.sh` in an editor and add the following:
+
+```
+#!/bin/sh -l
+
+# install flyctl
+curl -L https://fly.io/install.sh | sh
+
+# set flyctl environmental variables
+export FLYCTL_INSTALL="/home/travis/.fly"
+
+export PATH="$FLYCTL_INSTALL/bin:$PATH"
+
+echo "Successfully Installed Flyctl"
+
+# deploy app
+sh -c "flyctl deploy"
+
+# get app Information
+sh -c "flyctl info"
+
+exit 0
+```
+
+This command installs the Flyctl command-line tool and sets the environmental variables that make it possible to use the shortcut `flyctl` command. Next, it runs `flyctl deploy` to deploy the `flygreeting` application using the `fly.toml` file. Once the application is deployed, it runs `flyctl info` to give you information about the newly deployed app, including the name, version, status, and hostname.
+
+### Connect Travis CI to Your Git Repository
+Next, you need to link Travis with your repository hosting service. Visit [travis-ci.com](https://travis-ci.com/) and sign in with either GitHub or Bitbucket. You will be redirected to your repository host, where you can authorize Travis CI to have access to your GitHub or Bitbucket account.
+
+You will then be redirected back to Travis CI, where you will be able to select which repositories Travis CI has access to. Select the Flygreeting repository and continue with this tutorial.
+
+### Give Travis CI Access to Fly
+In order for Travis CI to automate deployment to Fly, you need to authorize it. You can do this by generating a Fly API token and adding it to Travis as an environment variable.
+
+Use Flyctl to generate an API token:
+
+```
+flyctl auth token
+```
+
+Copy the generated API code and head back to the Travis CI Dashboard. In the upper right corner, click your profile avatar and select [Settings](https://travis-ci.com/account/repositories). On the Repositories tab, you should see the `flygreeting` repository listed. Click the settings button in front of it.
+
+![](fly-travis-1.png)
+
+Navigate to the Environment Variables section and create a variable by setting `FLY_API_TOKEN` as the name, and the copied API key, as the value. Click the “Add” button to save this to your Travis account.
+
+![](fly-travis-2.png)
+
+### Testing the Deployment
+Now that your application is configured, your deployment script is written, and Travis is authorized to read your code repository, you’re ready to test the deployment process.
```

Comment:
Rewinds.... you really should be deploying from your own repo but we haven't forked the repo before cloning it yet so this is all working off the actual fly-example/flygreeting repo... for a normal user this would fail.